rs/embedders/benches/compilation.rs

```diff
@@ -103,6 +108,12 @@ fn generate_binaries() -> Vec<(String, BinaryEncodedWasm)> {
 
     result.push(("qrcode".to_string(), qrcode_wasm));
 
+    let mut decoder = libflate::gzip::Decoder::new(&GOVERNANCE_BENCH_CANISTER[..]).unwrap();
+    let mut buf = vec![];
+    decoder.read_to_end(&mut buf).unwrap();
+    let governance_wasm = BinaryEncodedWasm::new(buf);
+    result.push(("governance".to_string(), governance_wasm));
```

Comment:
This part now gets repeated a bunch. Maybe time to define a small helper function? I'm assuming we'll add some more benchmarks so it'll be handy.

Reply:
done